Code/Framework/AzFramework/Platform/Common/Xcb/azframework_xcb_files.cmake

```diff
@@ -6,6 +6,17 @@
 #
 #
 
+# generate Splash.h, based on Splash.png
+if(EXISTS ${PROJECT_SOURCE_DIR}/Resources/Splash.png)
+    execute_process(COMMAND ${CMAKE_COMMAND} -E make_directory ${CMAKE_BINARY_DIR}/AzFramework)
+    file(REMOVE ${CMAKE_BINARY_DIR}/AzFramework/Splash.h)
+    add_definitions(-DSPLASH_IMAGE_EXISTS)
+    execute_process(COMMAND xxd -i Splash.png ${CMAKE_BINARY_DIR}/AzFramework/Splash.h
```

Comment:
Splash.png is hardcoded, there is no way for the user to change it after compiling the engine.